packages/common/src/directives/ng_optimized_image/ng_optimized_image.ts

```diff
@@ -522,6 +557,34 @@ export class NgOptimizedImage implements OnInit, OnChanges, OnDestroy {
         !oversizedImage;
   }
 
+  private generatePlaceholder = (placeholderInput: string|boolean):
+      string|boolean|null => {
```

Comment:
nit: it can be a regular method on a class:
```suggestion
  private generatePlaceholder(placeholderInput: string|boolean):
      string|boolean|null {
```

Reply:
Fixed